### PR TITLE
Feature: Added new getUrlVariables API 

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by emdobrin — COMMENTED** [review]
Looks good, just couple minor comments

File: Sources/Identity.swift [hunk 1]
```diff
@@ -43,8 +44,16 @@ import Foundation
...
     public func readyForEvent(_ event: Event) -> Bool {
-        return state.bootupIfReady(getSharedState: getSharedState(extensionName:event:),
-                                   createXDMSharedState: createXDMSharedState(data:event:))
+        guard state.bootupIfReady(getSharedState: getSharedState(extensionName:event:),
+                                  createXDMSharedState: createXDMSharedState(data:event:)) else {
+            return false
+        }
+
+        if event.urlVariables {
+            return getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event, resolution: .lastSet)?.value != nil
```
Comment (emdobrin) [inline Sources/Identity.swift:53]: for this change we need to also bump the AEPCore dependency to 3.6.0 in AEPEdgeIdentity.podspec and Package.swift

File: Documentation/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  ```swift
+  // AppDelegate.swift
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    MobileCore.registerExtensions([Identity.self, Edge.self], {
+       MobileCore.configureWith(appId: "yourLaunchEnvironmentID")
+     })
+     ...
+  }
+  ```
```
Comment (kevinlind) [inline Documentation/getting-started.md:49]: Can you include Objective C examples too?

**Review by emdobrin — APPROVED** [review]

File: Documentation/getting-started.md [hunk 3]
```diff
@@ -27,23 +28,41 @@ The Adobe Experience Platform Identity for Edge Network extension has the follow
...
   ```swift
   // AppDelegate.swift
   import AEPCore
   import AEPEdge
   import AEPEdgeIdentity
   ```
 
-4. Register the Identity for Edge Extension with MobileCore:
-
   ```swift
```
Comment (kevinlind) [inline Documentation/getting-started.md:41]: nit: As these changes are in the same file, you can combine them both in the same "swift" language tag and just separate with "...".

**Review by kevinlind — APPROVED** [review]

File: Documentation/getting-started.md (hunk 3 above)
Comment (addb) [reply Documentation/getting-started.md:41]: We kept it separate so that the developer can copy paste it separately.

**Pull request merged**